velox/experimental/cudf/exec/Validation.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/experimental/cudf/CudfConfig.h"
+
+#include "velox/common/base/ExceptionHelper.h"
+
+namespace facebook::velox::cudf_velox {
```

Comment:
Curious, why is namespace named "cudf_velox" and not just "cudf"?

Reply:
Align with other cudf_velox class, I think it is because to distinguish the cudf library and cudf_velox library in Velox code